### ISSUE TITLE
Option to open a card from card browser in preview mode in stead of edit mode.

### ISSUE THREAD

**Issue by du5tmd** [ankidroid/Anki-Android#14425]
**Is your feature request related to a problem? Please describe.**

- Sometimes I want to just look up a card that is filled wirh cloze deletions or a table which is difficult to read in edit mode. I would then have to change it to preview mode and turn the card to see what I want.

**Describe the solution you'd like**

- It would be very nice if there is an option to let card browser open a card directly in preview mode. If there is a button to enter edit mode from it, that would be also great.

**Describe alternatives you've considered**

**Additional context**

**Comment by lukstbit** [comment]
CardBrowser has the option to preview in the menu(top app bar three dots menu).
You can preview all cards of a deck one by one or you can long click a single card and the use preview in menu. There's the option to go edit that card in the preview as well.
Isn't this what you ask?

**Comment by du5tmd** [comment]
Yes, I am aware of the current option to preview a card through menu. I just thought that it would be nice and faster if the number of steps to preview a card is reduced.

**Comment by lukstbit** [comment]
You can always tap the card to enter the editor and then in the note editor tap the preview icon in the app bar.
There will be some work in that part of the app but I'm not sure we can really improve the UI without making it too complex..

**Comment by du5tmd** [comment]
That is actually how I do it right now. Well I understand not all features can be implemented and it is better to keep the app less complex.

**Comment by BrayanDSO** [comment]
I'm okay with changing the browser preferred action, but we should improve its general UX before something like this

**Comment by david-allison** [comment]
I want this, but after the other changes for CardBrowser have been made

**Comment by 2uan2** [comment]
can I try working on this?

**Comment by 2uan2** [comment]
Would we want something like this in the browser options dialog to set the click action?

![Image](https://github.com/user-attachments/assets/e6372e09-a297-4a8c-b771-37f7d92e8471)

**Comment by Arthur-Milchior** [comment]
That seems all right to me
